lib/rubocop/cop/gemspec/required_ruby_version.rb

```diff
@@ -26,6 +28,12 @@ module Gemspec
       #     spec.required_ruby_version = '>= 2.6.0'
       #   end
       #
+      #   # bad (`required_ruby_version` set twice)
+      #   Gem::Specification.new do |spec|
+      #     spec.required_ruby_version = '>= 2.5.0'
+      #     spec.required_ruby_version = '>= 2.5.0'
+      #   end
```

Comment:
I think about it. It may be better to create a new cop like `Gemspec/DuplicateSpecificationAttribute` cop instead of this cop role extension. It seems preferable to keep one cop role simple based on Single Responsibility Principle :-) What do you think?

Reply:
I think it makes sense all together because the single responsibility here is checking `required_ruby_version`, and it basically was already doing this, but just for 0 and 1 calls, so now extended to 2+ as well. 

However, if you feel strongly about this I would be fine splitting it (and I’d move the check for 0 calls to that new cop too).

Reply:
IMHO, this cop is responsible for checking whether the values for `spec.required_ruby_version` of gemspec and `TargetRubyVersion` of .rubocop.yml match.

it seems simpler for another new cop to have the role of checking for duplicate attributes such as `required_ruby_version` for `Gem::Specification`. The new cop can check that various attributes such as `spec.name`, `spec.version`, `spec.description`, and etc are redundantly assigned.

Reply:
I agree with Koichi. Seems a separate cop looking for duplicated/empty properties would be a better idea.

Reply:
So there actually is already another cop that checks for duplicate assignments: `Gemspec/DuplicatedAssignment` 🤦